lf-mu/admin/class-lf-mu-admin.php

```diff
@@ -609,4 +609,147 @@ public function sync_projects() {
 		}
 
 	}
+
+	/**
+	 * Sync people data from https://github.com/cncf/people
+	 */
+	public function sync_people() {
+		$people_url = 'https://raw.githubusercontent.com/cncf/people/main/people.json';
+		$github_images_url = 'https://raw.githubusercontent.com/cncf/people/main/images/';
+
+		$args = array(
+			'timeout'   => 100,
+			'sslverify' => false,
+		);
+
+		$data = wp_remote_get( $people_url, $args );
+		if ( is_wp_error( $data ) || ( wp_remote_retrieve_response_code( $data ) != 200 ) ) {
+			return;
+		}
+		$people = json_decode( wp_remote_retrieve_body( $data ) );
+		foreach ( $people as $p ) {
+
+			$params = array(
+				'post_type'    => 'lf_person',
+				'post_title'   => $p->name,
+				'post_content' => $p->bio,
+				'post_status'  => 'publish',
+				'meta_input'   => array(),
+			);
+
+			if ( property_exists( $p, 'excerpt' ) ) {
+				$params['post_excerpt'] = $p->excerpt;
+			}
+			if ( property_exists( $p, 'company' ) ) {
+				$params['meta_input']['lf_person_company'] = $p->company;
+			}
+			if ( property_exists( $p, 'pronouns' ) ) {
+				$params['meta_input']['lf_person_pronouns'] = $p->pronouns;
+			}
+			if ( property_exists( $p, 'location' ) ) {
+				$params['meta_input']['lf_person_location'] = $p->location;
+			}
+			if ( property_exists( $p, 'linkedin' ) ) {
+				$params['meta_input']['lf_person_linkedin'] = $p->linkedin;
+			}
+			if ( property_exists( $p, 'twitter' ) ) {
+				$params['meta_input']['lf_person_twitter'] = $p->twitter;
+			}
+			if ( property_exists( $p, 'github' ) ) {
+				$params['meta_input']['lf_person_github'] = $p->github;
+			}
+			if ( property_exists( $p, 'wechat' ) ) {
+				$params['meta_input']['lf_person_wechat'] = $p->wechat;
+			}
+			if ( property_exists( $p, 'youtube' ) ) {
+				$params['meta_input']['lf_person_youtube'] = $p->youtube;
+			}
+			if ( property_exists( $p, 'priority' ) ) {
+				$params['meta_input']['lf_person_is_priority'] = $p->priority;
+			}
+			if ( property_exists( $p, 'image' ) ) {
+				if ( strpos( $p->image, 'http' ) !== false ) {
+					$image_url = $p->image;
```

Comment:
Should we do a https check here? If an image is embedded without https it will generate a mixed security warning.
https://stackoverflow.com/a/7468346/9042245
Maybe doesnt matter if images will only be on CNCF or pulled from GitHub directory.

Reply:
I think we want to have people add the images directly to GitHub rather than have dependencies on an arbitrary number of external sites.  So this may be something that the PR reviewer checks for on commits.